doc/deploy.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# Copyright 2013-2021 [The Foundry Visionmongers Ltd]
```

Comment:
[question] Are these two lines something we should be including in all our sources?

Reply:
Yeah. We need to make sure there is an Apache 2 licence, the `# SPDX-License-Identifier: Apache-2.0` was noted by a contact a the Linux Foundation as being satisfactory to indicate that license.

We should stick with the existing pattern we have in the `.py` files for now though (short Apache 2 string instead of SPDX).

Reply:
OK, I've added a review comment to my own PR to address this in a test file I added: https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/98/files#r758354644